app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -11,13 +11,18 @@
 .gem-c-pagination__item {
   @include govuk-font($size: 16, $line-height: (20 / 16));
   list-style: none;
+  margin-bottom: govuk-spacing(4);
+
+  &:last-child {
```

Comment:
Since there's presumably only ever going to be two items in this component, would it be simpler to just put the margin-bottom on the first child?